### ISSUE TITLE
No linebreaks in descriptions

### ISSUE THREAD

**Issue by mikhailshilkov** [pulumi/pulumi-google-native#137]
Discovery documents don't provide proper formatting of multi-line strings. Example from [here](https://github.com/pulumi/pulumi-google-native/blob/566c295a39fe8c3dd16e4a7894ff6de72423e5da/discovery/container_v1.json#L2794):

`The initial Kubernetes version for this cluster. Valid versions are those found in validMasterVersions returned by getServerConfig. The version can be upgraded over time; such upgrades are reflected in currentMasterVersion and currentNodeVersion. Users may specify either explicit versions offered by Kubernetes Engine or version aliases, which have the following behavior: - \"latest\": picks the highest valid Kubernetes version - \"1.X\": picks the highest valid patch+gke.N patch in the 1.X version - \"1.X.Y\": picks the highest valid gke.N patch in the 1.X.Y version - \"1.X.Y-gke.N\": picks an explicit Kubernetes version - \"\",\"-\": picks the default Kubernetes version`

It's clearly supposed to be a multi-line string with a bulleted list, like 

```
The initial Kubernetes version for this cluster. Valid versions are those found in validMasterVersions returned by getServerConfig. The version can be upgraded over time; such upgrades are reflected in currentMasterVersion and currentNodeVersion.

Users may specify either explicit versions offered by Kubernetes Engine or version aliases, which have the following behavior:

- "latest": picks the highest valid Kubernetes version
- "1.X": picks the highest valid patch+gke.N patch in the 1.X version
- "1.X.Y": picks the highest valid gke.N patch in the 1.X.Y version
- "1.X.Y-gke.N": picks an explicit Kubernetes version
- "","-": picks the default Kubernetes version
```

However, line breaks are missing.

**Comment by codyoss** [comment]
Yes I believe today linebreaks are stripped in the discovery doc format, but not in the [proto representation](https://github.com/googleapis/googleapis/blob/eb2c73b15704ce56eb93345fd14fb564c970a0a9/google/container/v1/cluster_service.proto#L1332-L1345). I will check in to see if this is intentional but my assumption is it was. Some languages/projects are particular about docstring line length. I know for our [generated Go clients](https://github.com/googleapis/google-api-go-client/blob/d285a80d164974b753ddb4dace224ba8b7d8d870/container/v1/container-gen.go#L855-L865) we set our own line breaks.
